### DISCUSSION TITLE
dv/v plot issue & SNR computing ...

### DISCUSSION THREAD

**Discussion by malmassri** [ROBelgium/MSNoise#315] [General]
Hello Thomas,

I was able to plot dv/v "msnoise cc dvv plot dvv", the result was the mean and median plot line. Similar to this graph demonstrated here "http://msnoise.org/doc/master/workflow_dvv/009b_plot_dvv.html". Sweet as. 

On the other hand, when I attempted to plot for a specific station pair obviously, the mean and median will alter now but I expected to see a third line for the station pair, but this did not materialize.    Am I wrong somewhere? Your help is highly appreciated. 
>> I run "msnoise cc dvv plot dvv -p NZ_COVZ_NZ_FWVZ".

Both Coherence and SNR depend on time lags, which can assist in determining the appropriate coherence threshold. I am aware that coherence is output when MWCS calculations are carried out. However, I searched for SNR inside the MSnoise codes, but I was unable to find it. Have you taken into account the SNR computation as part of the MSNoise processes? 

Thank you in advance.

**Comment by ThomasLecocq** [comment]
humm; the pair-plotting thingy has not been ported to msnoise2 I think ... will have to investigate that!
